### ISSUE TITLE
Compiler warnings seem to block submission

### ISSUE THREAD

**Issue by oml1111** [appirio-tech/arena-web#738]
![compilation](https://user-images.githubusercontent.com/9612370/27735048-471b7c16-5da6-11e7-9355-711d0524f12e.PNG)

![submission](https://user-images.githubusercontent.com/9612370/27735053-4c8cc40c-5da6-11e7-9dac-91bde4fcd547.PNG)

**Comment by square1001** [comment]
Yes. I asked to admin, and it is intended to do so, and it's not a bug.
